requirements.txt

```diff
@@ -14,3 +14,5 @@ Pillow
 Faker
 mypy
 django-stubs
+social-auth-app-django
+psycopg2
```

Comment:
Postgress is not a dependency of our app.

Reply:
:+1:

Reply:
social-auth-app-django won't start without this library. It may be related to the SOCIAL_AUTH_POSTGRES_JSONFIELD setting but I've haven't tested it.

Reply:
Please test it then and set this setting in prod. The project should also work on sqlite

Reply:
@cytadela8 this works fine in sqlite.